listings_app/urls.py

```diff
@@ -16,6 +16,9 @@
 from django.contrib import admin
 from django.urls import path
 
+from . import views
+
 urlpatterns = [
     path('admin/', admin.site.urls),
+    path('', views.homepage, name='homepage')
```

Comment:
normally the path would be `/`

Reply:
hey @lili2311, thanks for your comments. About '/', I  thought the same, but when I added it I was said by VS Code that it was deprecated, that it wasn't needed anymore. Do you want me to add it anyway?